templates/advanced/test/unit/tags/components/example.test.js

```diff
@@ -0,0 +1,56 @@
+const { expect } = require('chai');
+
+const { ExampleComponent } = require('../../../../src/tags/components/example/example');
+
+describe('ExampleComponent', () => {
+  describe('General', () => {
```

Comment:
Maybe add a `beforeEach` to the top which creates an instance of their `ExampleComponent` so that they can re-use it in each test

Reply:
Implemented.